### PR TITLE
file to Renamed react-native-gradle-plugin to @react-native/gradle-pl…

### PR DESCRIPTION
**Pull request by rahul3002**
…ugin

## Changelog

[CATEGORY] [TYPE] - Message

### CODE REVIEW

File: .circleci/verdaccio.yml [hunk 1]
```diff
@@ -29,7 +29,7 @@ packages:
   'react-native-codegen':
     access: $all
     publish: $all
-  'react-native-gradle-plugin':
+  'react-native/gradle-plugin.':
     access: $all
     publish: $all
```
Comment (cortinico) [inline .circleci/verdaccio.yml:34]: You can remove this block entirely

File: .gitignore [hunk 2]
```diff
@@ -24,7 +24,7 @@ project.xcworkspace
 
 # Gradle
 /build/
-/packages/react-native-gradle-plugin/build/
+/packages/react-native/gradle-plugin./build/
```
Comment (cortinico) [inline .gitignore:27]:
```suggestion
/packages/react-native/gradle-plugin/build/
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -10,7 +10,7 @@
 ### Changed
 
 - Accept TypeScript type `T | null | undefined` as a maybe type of T in turbo module. ([9ecd203eec](https://github.com/facebook/react-native/commit/9ecd203eec97e7d21d10311d950c9f8f30c7a4b1) by [@ZihanChen-MSFT](https://github.com/ZihanChen-MSFT))
-- Bump react-native-gradle-plugin to 0.70.3 ([e33633644c](https://github.com/facebook/react-native/commit/e33633644c70ea39af6e450fcf31d9458051fd5f) by [@dmytrorykun](https://github.com/dmytrorykun))
+- Bump react-native/gradle-plugin. to 0.70.3 ([e33633644c](https://github.com/facebook/react-native/commit/e33633644c70ea39af6e450fcf31d9458051fd5f) by [@dmytrorykun](https://github.com/dmytrorykun))
```
Comment (cortinico) [inline CHANGELOG.md:13]: Let's not edit the CHANGELOG

File: build.gradle.kts [hunk 4]
```diff
@@ -34,7 +34,7 @@ allprojects {
 
 tasks.register("cleanAll", Delete::class.java) {
   description = "Remove all the build files and intermediate build outputs"
-  dependsOn(gradle.includedBuild("react-native-gradle-plugin").task(":clean"))
+  dependsOn(gradle.includedBuild("react-native/gradle-plugin.").task(":clean"))
```
Comment (cortinico) [inline build.gradle.kts:37]: You should not edit this line as the path is not changing

File: build.gradle.kts [hunk 5]
```diff
@@ -53,7 +53,7 @@ tasks.register("cleanAll", Delete::class.java) {
 
 tasks.register("buildAll") {
   description = "Build and test all the React Native relevant projects."
-  dependsOn(gradle.includedBuild("react-native-gradle-plugin").task(":build"))
+  dependsOn(gradle.includedBuild("react-native/gradle-plugin.").task(":build"))
```
Comment (cortinico) [inline build.gradle.kts:56]: You should not edit this line as the path is not changing

File: build.gradle.kts [hunk 6]
```diff
@@ -67,7 +67,7 @@ tasks.register("buildAll") {
 
 tasks.register("downloadAll") {
   description = "Download all the depedencies needed locally so they can be cached on CI."
-  dependsOn(gradle.includedBuild("react-native-gradle-plugin").task(":dependencies"))
+  dependsOn(gradle.includedBuild("react-native/gradle-plugin.").task(":dependencies"))
```
Comment (cortinico) [inline build.gradle.kts:70]: You should not edit this line as the path is not changing

File: package.json [hunk 7]
```diff
@@ -128,7 +128,7 @@
     "pretty-format": "^26.5.2",
     "promise": "^8.2.0",
     "react-devtools-core": "^4.26.0",
-    "react-native-gradle-plugin": "^0.71.4",
+    "react-native/gradle-plugin.": "^0.71.4",
```
Comment (cortinico) [inline package.json:131]:
```suggestion
    "react-native/gradle-plugin": "^0.71.4",
```

File: packages/react-native-gradle-plugin/README.md [hunk 8]
```diff
@@ -1,4 +1,4 @@
-# react-native-gradle-plugin
+# react-native/gradle-plugin.
```
Comment (cortinico) [inline packages/react-native-gradle-plugin/README.md:1]:
```suggestion
# @react-native/gradle-plugin
```

File: packages/react-native-gradle-plugin/README.md [hunk 9]
```diff
@@ -7,10 +7,10 @@ A Gradle Plugin used to support development of React Native applications for And
 ## Installation
 
 ```
-yarn add react-native-gradle-plugin
+yarn add react-native/gradle-plugin.
```
Comment (cortinico) [inline packages/react-native-gradle-plugin/README.md:10]:
```suggestion
yarn add @react-native/gradle-plugin
```

File: packages/react-native-gradle-plugin/package.json [hunk 10]
```diff
@@ -1,12 +1,12 @@
 {
-  "name": "react-native-gradle-plugin",
+  "name": "react-native/gradle-plugin.",
```
Comment (cortinico) [inline packages/react-native-gradle-plugin/package.json:2]:
```suggestion
  "name": "@react-native/gradle-plugin",
```

This needs to be fixed all over your PR

**Comment by hoxyq** [comment]
Oops, looks like it automatically closed after my push and right now I am unable to push to `rahul3002/main`
I've pushed just `main` branch by mistake, so there is no commits, thats why it probably closed

**Pull request closed without merging**